src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -10,47 +12,86 @@ const localize            = require('../../../../_common/localize').localize;
 const showLoadingImage    = require('../../../../_common/utility').showLoadingImage;
+const TabSelector         = require('../../../../_common/tab_selector');
 
 const Authenticate = (() => {
-    let is_action_needed     = false;
     let is_any_upload_failed = false;
     let file_checks          = {};
-    let $button,
+    let onfido,
+        $button,
         $submit_status,
         $submit_table;
 
-    const onLoad = () => {
-        BinarySocket.send({ get_account_status: 1 }).then((response) => {
-            $('#loading_authenticate').remove();
-            if (response.error) {
-                $('#error_message').setVisibility(1).text(response.error.message);
-            } else {
-                const status = response.get_account_status.status;
-                is_action_needed = /document_needs_action/.test(response.get_account_status.status);
-                if (!/authenticated/.test(status)) {
-                    init();
-                    const language            = getLanguage();
-                    const language_based_link = ['ID', 'RU', 'PT'].includes(language) ? `_${language}` : '';
-                    const $not_authenticated  = $('#not_authenticated');
-                    $not_authenticated.setVisibility(1);
-                    let link = Url.urlForCurrentDomain(`https://marketing.binary.com/authentication/Authentication_Process${language_based_link}.pdf`);
-                    if (Client.isAccountOfType('financial')) {
-                        $('#not_authenticated_financial').setVisibility(1);
-                        link = Url.urlForCurrentDomain('https://marketing.binary.com/authentication/MF_Authentication_Process.pdf');
-                    }
-                    $not_authenticated.find('.learn_more').setVisibility(1).find('a').attr('href', link);
+    const simulateCallForApiKey = () => new Promise((resolve, reject) => {
+        if (Cookies.get('onfido_token')) {
+            resolve(Cookies.get('onfido_token'));
+        } else {
+            let token = '';
 
-                    if (isIdentificationNoExpiry(Client.get('residence'))) {
-                        $('#expiry_datepicker_proofid').setVisibility(0);
-                        $('#exp_date_2').datepicker('setDate', '2099-12-31');
-                    }
-                } else if (!/age_verification/.test(status)) {
-                    $('#needs_age_verification').setVisibility(1);
-                } else {
-                    $('#fully_authenticated').setVisibility(1);
-                }
+            if (window.location.host === 'localhost') {
+                token = 'eyJhbGciOiJIUzI1NiJ9.eyJwYXlsb2FkIjoiNHZlS0ROeEdQMWoyTUV4QlBQQnVXSklxMDM0akpiWjBNTGdvK3M1YjNRd1NtOXdKODkwb29oVkFBa3B1XG4wM2w1Y1UwNEpETTduaU5hU3BqQitoUUVkalVEYk9abmdLU25yaTNERlZlbFVXcz1cbiIsInV1aWQiOiJIeHNhMDl1dmtBOCIsImV4cCI6MTU2MzMzNzg1MH0.ofMCHCFqPNeJedp2Z_U6x1FdCnhVkSkDdVyYJfPScIc';
+                resolve(token);
```

Comment:
is this going to be removed later? do we need a comment?

Reply:
yes, this will be removed later, ill put comment, thank you 🙇